ecommerce/static/templates/_offer_course_list.html

```diff
@@ -50,6 +54,13 @@ <h4 class="pagination-range pull-right hidden-xs">Showing <%= (courses.lowerLimi
                                 </p>
                             </div>
                             <% } else { %>
+                                <% if (course.attributes.special_coupon) { %>
+                            <div class="pull-left">
+                                <p class="course-new-price">
+                                    <span><%- course.attributes.special_coupon_message %> $<%= course.attributes.new_price %></span>
+                                </p>
+                            </div>
+                                <% } else { %>
```

Comment:
Bad indentation; I believe parentheses are not necessary. @Jacatove

Reply:
Parentheses are required. @Squirrel18